src/utils/sdk-registry.ts

```diff
@@ -0,0 +1,96 @@
+import { Address } from "viem";
+
+function getSDKRegistry() {
+  try {
+    const sdkPath = require.resolve("@rhinestone/sdk");
+    const registryPath = sdkPath.replace(
+      "/dist/src/index.js",
+      "/dist/src/orchestrator/registry.js"
+    );
+    return require(registryPath);
+  } catch (error) {
+    throw new Error(
+      `Failed to load SDK registry: ${
+        error instanceof Error ? error.message : "Unknown error"
+      }`
+    );
+  }
+}
+
+const registry = getSDKRegistry();
+
+// export type BundleResult = any;
+// export type OrderPath = any;
+
+export type BundleResult = {
```

Comment:
These feel like they maybe deserve their own file - rather than being in the `sdk-registry.ts`. Thoughts?